static/src/javascripts/projects/commercial/modules/third-party-tags/plista-outbrain-renderer.spec.js

```diff
@@ -47,7 +40,6 @@ describe('Plista Outbrain renderer', () => {
             config.set('switches.plistaForOutbrainAu', true);
             config.set('page.edition', edition);
             initPlistaOutbrainRenderer().then(() => {
```

Comment:
Can we rename this to `initPlistaRenderer()`?

Reply:
Done